LaTeX/modelica.cls

```diff
@@ -121,3 +126,6 @@
 \lstset{language = modelica,
        basicstyle=\fontsize{9pt}{10.5pt}\ttfamily,
        backgroundcolor = \color{white}}
+\RequirePackage{cleveref}      %% For easier cross-references
```

Comment:
why not use `\autoref` from the (already loaded?!) hyperref package.

Reply:
I could change it to use and recommend `autoref` instead since we always capitalize the names, but in publications where you write "section 1.1", you cannot really use autoref commands to start a sentence... So I always check for cleveref when writing references these days.

Reply:
True, but I always redefine the capitalisation for autoref with: 
```
%% Adjust the formatting for refrences if autoref is used.
  \AtBeginDocument{\def\equationautorefname~#1\null{Equation~(#1)\null}}
  \AtBeginDocument{\def\chapterautorefname~#1\null{Chapter~#1\null}}
  \AtBeginDocument{\def\sectionautorefname~#1\null{Section~#1\null}}
```

Reply:
I don't say you have to do that, but I normally like to keep the dependencies on other packages low if possible.

Reply:
Already did it. Smaller change.